ui/main/src/app/modules/feed/components/card-list/card-list.component.scss

```diff
@@ -28,6 +28,24 @@
   margin-right: 20px;
   margin-left: 1px;
   border-color:var(--opfab-card-tab-border-color);
+}
+
+.opfab-ack-all-cards {
+  background-color: var(--opfab-bgcolor);
+  text-align: center;
+  font-size: 13px;
+  padding-top : 5px;
+}
```

Comment:
add a padding-right : 20px ;  --> to center precisely the text with lightcard

Reply:
Done.